catalyst-gateway/bin/src/db/event/mod.rs

```diff
@@ -0,0 +1,237 @@
+//! Catalyst Election Database crate
+use std::{
+    str::FromStr,
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        Arc, OnceLock,
+    },
+};
+
+use bb8::Pool;
+use bb8_postgres::PostgresConnectionManager;
+use tokio_postgres::{types::ToSql, NoTls, Row};
+use tracing::{debug, debug_span, error, Instrument};
+
+use crate::settings::Settings;
+
+pub(crate) mod error;
+pub(crate) mod legacy;
+pub(crate) mod schema_check;
+
+/// Database version this crate matches.
+/// Must equal the last Migrations Version Number.
```

Comment:
Maybe add where is this 'Migrations Version Number' come from

Reply:
Done